validator/remote/src/integration-test/java/tech/pegasys/teku/validator/remote/typedef/OkHttpValidatorTypeDefClientTest.java

```diff
@@ -410,6 +414,33 @@ private StateValidatorData generateStateValidatorData() {
         validator);
   }
 
+  @TestTemplate
+  public void postAttesterDuties_WhenSuccess_ReturnsResponse() throws JsonProcessingException {
+    final List<AttesterDuty> duties = List.of(randomAttesterDuty(), randomAttesterDuty());
+    final AttesterDuties response =
+        new AttesterDuties(true, dataStructureUtil.randomBytes32(), duties);
+
+    final String body = serialize(response, ATTESTER_DUTIES_RESPONSE_TYPE);
+    mockWebServer.enqueue(new MockResponse().setResponseCode(SC_OK).setBody(body));
+
+    Optional<AttesterDuties> result =
+        okHttpValidatorTypeDefClient.postAttesterDuties(UInt64.ONE, IntSet.of(1, 2));
+
```

Comment:
The extra thing we can do here is use `RecordedRequest` to get the actual request made to the mockWebServer (takeRequest) and then compare the body of that request to what our expectation was...